lib/matplotlib/backend_tools.py

```diff
@@ -835,9 +835,9 @@ def _release(self, event):
             twinx, twiny = False, False
             if last_a:
                 for la in last_a:
-                    if a.get_shared_x_axes().joined(a, la):
+                    if a._shared_axes["x"].joined(a, la):
```

Comment:
As above, don’t use the private API if not necessary.

Reply:
ditto.